btree/btree.c

```diff
@@ -2489,15 +2424,17 @@ static void ff_fini(const struct nd *node)
 
 static int ff_count(const struct nd *node)
 {
-	int used = ff_data(node)->ff_used;
-	if (ff_data(node)->ff_level > 0)
+	struct ff_head *h = ff_data(node);
+	int used = h->ff_used;
```

Comment:
I do not see any issue with previous code, why it is changed?

Reply:
This particular variable "struct ff_head *h" was added to so that segaddr data can be assigned to it.
the code was updated to: struct ff_head *h =  segaddr_addr(addr);
On reversing those changes, I just replaced "segaddr_addr(addr)" with "ff_data(node)"
Incase in future, we again decide to go with segaddr, it will be easy to update.

Reply:
In that code also you could have just replaced ff_data with segaddr_addr.

Reply:
Ok. I will update it!